### PR TITLE
upd(freetube-deb): `0.16.0` -> `0.17.1`

### PR DESCRIPTION
**Pull request by weevilgit**

### CODE REVIEW

**Comment by Henryws** [comment]
Fix the title to be `` upd(freetube-deb): `0.16.0` -> `0.17.1` ``

**Review by Henryws — CHANGES_REQUESTED** [review]
Update `hash`.

File: packages/freetube-deb/freetube-deb.pacscript [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name="freetube-deb"
 gives="freetube"
-version="0.16.0"
+version="0.17.1"
 url="https://github.com/FreeTubeApp/FreeTube/releases/download/v${version}-beta/${gives}_${version}_amd64.deb"
 description="An Open Source YouTube app for privacy."
 hash="e5e7911d8e673796a3adaa1099861c883b3ea6b520065e6e209688d5b1241494"
```
Comment (Henryws) [inline packages/freetube-deb/freetube-deb.pacscript:6]:
```suggestion
hash="0abdf750e2589a8918bb3f04bc72e358f32f699793ef703fbb74c04c4ef783af"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by weevilgit** [comment]
> Fix the title to be `` upd(freetube-deb): `0.16.0` -> `0.17.1` ``
Done!




> Update `hash`.
Thanks! Done.

**Pull request merged**